src/test/java/edu/RobertoAranovich/advancedjava/AuthorTest.java

```diff
@@ -0,0 +1,87 @@
+package edu.RobertoAranovich.advancedjava;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Unit test for Author class.
+ *
+ * @author Roberto Aranovich
+ */
```

Comment:
this is old style junit (pre 4) which is fine. 
but going forward please use junit 4 @test etc. annotations.

Reply:
Done